GoogleForCreators/web-stories-wp
# Redesign: ask for confirmation when deleting global presets.

## Summary
When deleting a global preset, confirmation is asked first.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Ensure some Saved styles exist and some Saved colours exist -- both global and local.
2. Delete a local colour for the current story. Verify it's deleted correctly.
3. Click to delete a global colour. Verify a confirmation is asked first and in case of approval, it's deleted.
4. Verify the same for Saved style -- it's asking a confirmation first.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6070
